gitman/tests/test_shell.py

```diff
@@ -63,21 +64,25 @@ def test_ln(self, mock_call):
     def test_ln_missing_parent(self, mock_call):
         """Verify the commands to create symbolic links (missing parent)."""
         shell.ln('mock/target', 'mock/source')
-        assert_calls(mock_call, ["ln -s mock/target mock/source"])
+        assert_calls(mock_call, ["mkdir -p mock",
```

Comment:
But ln is not supported on windows, right ? So this test is not played on windows.
So dont need to write a specific test for windows